cfme/common/vm.py

```diff
@@ -32,8 +32,8 @@
 retire_remove_button = "//span[@id='remove_button']/a/img|//a/img[contains(@src, '/clear')]"
 
 set_ownership_form = Form(fields=[
-    ('user_name', Select("//select[@id='user_name']")),
-    ('group_name', Select("//select[@id='group_name']")),
+    ('user_name', AngularSelect('user_name')),
```

Comment:
@psav but does we support old versions? I thought for now we supporting 5.6+ and upstream for tests